tests/spread/integration/apache2/task.yaml

```diff
@@ -0,0 +1,23 @@
+summary: Integration tests for apache2
+
+execute: |
+  rootfs="$(install-slices apache2_bins base-passwd_data util-linux_cli-helpers)"
```

Comment:
Do we still need `util-linux_cli-helpers`? I think we only added it temporarily to get the `namei` utility to debug?